### ISSUE TITLE
How to make top menu clickable of drop down menu

### ISSUE THREAD

**Issue by imalisiddique** [wp-bootstrap/wp-bootstrap-navwalker#9]
I created website with  wp-bootstrap-navwalker ,but how can i create top menu clickable .
http://newbury.viprecipe.com/01/
Here is my link
i want to family support and childcare clickable

**Comment by robhadfield** [comment]
I would rephrase the menu drop item and add those items to the dropdown list. Either that or replace the click to show with a hover... of course then you have mobile to worry about :)

For clarity:
- Child Care
  - Child care information
  - Pre school
  - Picture

etc...

**Comment by imalisiddique** [comment]
could you provide me any code when i click childcare then it open it's page

**Comment by robhadfield** [comment]
This might help you : https://github.com/320press/wordpress-bootstrap/issues/62

**Comment by SideQuestDIY** [comment]
As I mentioned in my reply on issue #8

> The parent menu item is not clickable as a link because that is how things are done with the bootstrap framework. The reason for this is that bootstrap dropdowns don't pop open on hover as most people are used to, but on click. This is so that the menus system is usable on mobile devices such as tablets and cellphones. If you change this you will not be able to access sub-menu items on a mobile device.

As making this change will create more issues than it solves, (although it is definitely possible to achieve) it is not something that this script will support.

@Rob-Che's recommendation of adding your page to the dropdown is the best all round solution.
